config.mak.uname

```diff
@@ -335,6 +335,16 @@ endif
 ifeq ($(uname_S),Windows)
 	GIT_VERSION := $(GIT_VERSION).MSVC
 	pathsep = ;
+	# Prepend MSVC 64-bit tool-chain to PATH.
+	#
+	# TODO A normal git-bash window DOES NOT contain the path to CL.EXE
+	# TODO and friends in $PATH.
+	# TODO
+	# TODO Warning: There is a MS link.exe next to CL.exe and a gcc link.exe
+	# TODO next to gcc.exe, so we have to prepend this onto the existing
+	# TODO $PATH.  (Or fix clink.pl and lib.pl to use full paths.)
```

Comment:
Prepending is the right thing to do, I am convinced by this argument. Let's skip the part included in the parentheses.